app/assets/stylesheets/course.css.scss

```diff
@@ -14,4 +12,23 @@
     left:50%;
     margin-left:-5em;
   }
+  
+  .categories {
+    margin-top:10px;
+
+    .list-group-item {
+      padding:2px 0px;
```

Comment:
Colon after property should be followed by one space<br>`0px` should be written without units as `0`